pkgs/wine/supportFlags.nix

```diff
@@ -37,6 +37,7 @@ rec {
     faudioSupport = false;
     vkd3dSupport = false;
     embedInstallers = false;
+    waylandSupport = false;
```

Comment:
Does this affect the vkd3d functionality? Should we extend this PR to also include the `wayland` modifications to the `wine` builder in Nixpkgs?

Reply:
The builder requires waylandSupport to be present in the supportFlags now. I set it to false because the wine-tkg derivation creates a staging version of wine and upstream nixpkgs does not allow that with wayland support (yet)